src/Http/Controllers/AppointmentController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace LibreEHR\FHIR\Http\Controllers;
+
+use Illuminate\Http\Request;
+use LibreEHR\FHIR\Adapters\FHIRAppointmentAdapter;
+use Illuminate\Support\Facades\DB;
+use phpDocumentor\Reflection\DocBlock\Tags\Return_;
+
+class AppointmentController extends AbstractController
+{
+    public function __construct()
+    {
+        $this->adapter = new FHIRAppointmentAdapter;
+    }
+
+    public function index()
+    {
+
+    }
+
+    public function getSchedule()
+    {
+        $results = DB::select('select * from openemr_postcalendar_events');
+        $json = json_encode($results);
+
+        return $json;
+    }
+
+    public function addSlot( Request $request )
+    {
+        $start = $request->input('start');
```

Comment:
Also Slot should be its own Resource endpoint with its own controller and adapter.